docs/user/api/cross-site-requests.rst

```diff
@@ -0,0 +1,30 @@
+Cross-site requests
+===================
+
+Cross site requests are allowed for the following endpoints:
+
+- :ref:`/api/v2/footer_html/ <api/v2:Undocumented resources and endpoints>`
+- :ref:`/api/v2/search/ <server-side-search/api:API V2 (deprecated)>`
+- :ref:`/api/v2/embed/ <api/v2:Embed>`
+- :ref:`/api/v2/sustainability/ <api/v2:Undocumented resources and endpoints>`
+- :ref:`/api/v3/embed/ <api/v3:Embed>`
+
+Except for the sustainability API, all of the above endpoints
+don't allow you to pass credentials in cross-site requests.
+This means you can make use of those APIs to get information from public versions only.
+
+Cookies
+-------
+
+For |org_brand| our session cookies have the ``SameSite`` attribute set to ``None``,
+this means they can be sent in cross site requests.
+This is needed for our sustainability API only,
+to not show ads if the current user is a :ref:`Gold User <advertising/ad-blocking:Going ad-free>`.
```

Comment:
Should we add to the explanation that it's _also_ necessary to allow custom domains because we allow custom domains?

Reply:
Not sure I understand, do you mean like it's necessary to use the cookies from custom domains?

Reply:
Hmm, sentence was weird. When you say:

> do you mean like it's necessary to use the cookies from custom domains?

I think what I mean is this:

> it's necessary to set `SameSite` to `None` on certain cookies because there are custom domains [and sustainability API is not available on custom domains]

Sorry that I can't come up with a good direct suggestion. But custom domains are a fundamental reason for this, we should perhaps mention it... and if not, we shouldn't :)

Reply:
Btw, this is not like a critical comment, just wondering if you would agree.. I can put it in a note inside the documentation for next visit :+1:

Reply:
oh, this is for docs pages in general, since they are served from a different domain than the dasbboard. When we refer to custom domains, we usually mean the "custom domain" option we have, this is, serving a page from something else other than `readthedocs.io`

Reply:
Ah of course - I thought that you could somehow instruct the cookie about which sites it is eligible for, but I see it's not possible, it supports just None/Lax/SameSite etc :+1: